test/integration/vpc_sc_project/controls/vpc_sc_project.rb

```diff
@@ -0,0 +1,91 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id                         = attribute('project_id')
+project_number                     = attribute('project_number')
+service_account_email              = attribute('service_account_email')
+compute_service_account_email      = attribute('compute_service_account_email')
+group_email                        = attribute('group_email')
+group_name                         = attribute('group_name')
+
+control 'project-factory-vpc-sc-project' do
```

Comment:
We should add a test which validates presence in the perimeter.

Reply:
Added